pkg/ccl/backupccl/manifest_handling.go

```diff
@@ -1098,8 +1098,11 @@ func readEncryptionOptions(
 	ctx context.Context, src cloud.ExternalStorage,
 ) ([]jobspb.EncryptionInfo, error) {
 	files, err := getEncryptionInfoFiles(ctx, src)
+	wrappedError := `could not find or read encryption information. 
```

Comment:
I'd prefer if we did this in the following way:

let readEncryptionOptions returned a marked error when it is unable to find the encryption info file, you can checkout `errors.Mark`. The callsite in SHOW BACKUP that calls readEncryptionOptions should then check for this mark, and issue an `errors.WithHint`. This method is used by more than just show and so I'd prefer not having this NOTE in this shared path.

Reply:
done. interestingly, the marked error does not get wrapped; hence the gymnastics in manifest_handling.go.